server/budgettracker.web/Properties/launchSettings.json

```diff
@@ -18,7 +18,7 @@
     "budgettracker.web": {
       "commandName": "Project",
       "launchBrowser": true,
-      "applicationUrl": "https://localhost:5001;http://localhost:5000",

```

Comment:
Is this the workaround you had to do to get postman to work? That's weird. I haven't had any problems. Maybe it's different between what OS dotnet core is running on?

FYI this is not a request to change so you can resolve this when you see it.

Reply:
I probably didn't mean to commit this, it should make a difference my changes to I can undo this.